content/developer/reference/backend/views.rst

```diff
@@ -178,6 +178,13 @@ an impact on all view types.
 
 .. todo:: Views main content section, with field, group & separator ?
 
+* ``class``
+  optional string containing some classes to set on the root node of the view.
+
+* ``js_class``
+  optional component class to use to render the view. If set, the class ``o_'js_class'_view``
```

Comment:
don't know how to write this correctly

custom_Kanban gives o_custom_kanban_view